java/src/test/java/com/google/oak/client/OakClientTest.java

```diff
@@ -81,14 +103,18 @@ public void close() throws Exception {
 
   /** This test demonstrates the use of the {@code com.google.oak.client.OakClient} API. */
   @Test
-  public void testOakClient() {
+  public void testOakClient() throws Exception {
     Result<OakClient<TestTransport>, Exception> oakClientCreateResult =
         OakClient.create(new TestTransport(), new InsecureAttestationVerifier());
     assertTrue(oakClientCreateResult.isSuccess());
     OakClient<TestTransport> oakClient = oakClientCreateResult.success().get();
 
-    Result<byte[], Exception> oakClientInvokeResult = oakClient.invoke(TEST_REQUEST);
-    assertTrue(oakClientInvokeResult.isSuccess());
-    assertArrayEquals(oakClientInvokeResult.success().get(), TEST_RESPONSE);
+    for (int i = 0; i < TEST_SESSION_SIZE; i++) {
+      Result<byte[], Exception> oakClientInvokeResult = oakClient.invoke(TEST_REQUEST);
+      assertTrue(oakClientInvokeResult.isSuccess());
+      System.out.print(oakClientInvokeResult.success().get().length);
+      assertArrayEquals(oakClientInvokeResult.success().get(), TEST_RESPONSE);
+    }
+    oakClient.close();
```

Comment:
Can you use `try-with-resource` here, so you don't have to explicitly call `close`?

For instance something like:
```
try (OakClient<TestTransport> oakClient = oakClientCreateResult.success().get()) {
 for(...) {...}
}
```